overlays/host/etc/dhcp/dhcpd.conf.ww

```diff
@@ -20,15 +20,11 @@ option architecture-type   code 93  = unsigned integer 16;
 if exists user-class and option user-class = "iPXE" {
     filename "http://{{$.Ipaddr}}:{{$.Warewulf.Port}}/ipxe/${mac:hexhyp}";
 } else {
-    if option architecture-type = 00:0B {
-        filename "/warewulf/arm64.efi";
-    } elsif option architecture-type = 00:09 {
-        filename "/warewulf/x86_64.efi";
-    } elsif option architecture-type = 00:07 {
-        filename "/warewulf/x86_64.efi";
-    } elsif option architecture-type = 00:00 {
-        filename "/warewulf/x86_64.kpxe";
-    }
+{{range $type,$name := $.Tftp.IpxeBinaries }}
```

Comment:
We also need to update `/etc/warewulf/examples/static-dhcpd.conf.ww`

Reply:
Actually, it's looking like that example file should just be removed, as it looks like the static functionality was already merged into the overlay with a variable.

Reply:
Worth noting that these were "elsif" before, and now it's just a list of ifs. Probably fine.